SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -0,0 +1,175 @@
+// REQUIRES: xptifw
+// RUN: %clangxx -fsycl %s -DXPTI_COLLECTOR -shared -fPIC -std=c++17 -o %t_collector.dll -lxptifw
+// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env XPTI_TRACE_ENABLED=1 env XPTI_FRAMEWORK_DISPATCHER=%xptifw_lib env XPTI_SUBSCRIBERS=%t_collector.dll %CPU_RUN_PLACEHOLDER %t %CPU_CHECK_PLACEHOLDER
```

Comment:
The test is running on CPU only. Is it a requirement? If so, `cpu` should be added to REQUIRES. If not we can remove CPU_RUN_PLACEHOLDERs.

As for my understanding, it should work for any BE so it is better to removed dependencies on specific device type.